fda-model/paireddata/StepwisePairedData.cs

```diff
@@ -6,8 +6,8 @@ namespace paireddata
     public class StepwisePairedData: IPairedData
     {
 
-        public double[] Xvals { get; set; }
```

Comment:
now they can only be constructed via the constructor and accessed via the getter. I like that. I think i would still like to have a private final array of double named _x that we return via xvals, it adds "boiler plate" but it gives us a bit finer control.